README.md

```diff
@@ -192,6 +192,9 @@ A `Micro::Case::Result` stores the use cases output data. These are their main m
 - `#type` a Symbol which gives meaning for the result, this is useful to declare different types of failures or success.
 - `#data` the result data itself.
 - `#[]` and `#values_at` are shortcuts to access the `#data` values.
+- `#fetch` and `#fetch_values` are another way of accessing values of the result data, but raises a `KeyError` if the one of the keys are not present in the result.
+- `#keys` returns an array of keys present in the result data.
+-  returns an array of values of the result data, and if one
```

Comment:
@tomascco I believe that this line must be removed.

Reply:
Yes, you're right, it was a typo :/